packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1329,6 +1330,28 @@ export default class NewVm extends BaseComponent {
 
   // DISKS -----------------------------------------------------------------------
 
+  _srsNotOnSameHost = createSelector(
+    () => this.props.srs,
+    () => this.state.state.existingDisks,
```

Comment:
We are testing the compatibility between `existingDisks`, the compatibility between `VDIs` and the compatibility between  them. Why not creating a selector which merge them? Then the CB need just to test the compatibility between the got vdis.

Reply:
it's done.